module/documents/activity/mixin.mjs

```diff
@@ -1045,6 +1045,11 @@ export default function ActivityMixin(Base) {
             width: 400,
             top: config.event ? config.event.clientY - 80 : null,
             left: window.innerWidth - 710
+          },
+          window: {
+            title: game.i18n.localize(`DND5E.${this.type === "heal" ? "Healing" : "Damage"}Roll`),
```

Comment:
You should just `this.damageFlavor` for this. You can change it for the heal activity to use `DND5E.HealingRoll` rather than just `DND5E.Healing`.